stdlib/generate/from_test.go

```diff
@@ -0,0 +1,122 @@
+package generate_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/flux"
+	_ "github.com/influxdata/flux/builtin"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/querytest"
+	"github.com/influxdata/flux/runtime"
+	_ "github.com/influxdata/flux/stdlib"
```

Comment:
I think you only need the `"github.com/influxdata/flux/builtin"` line above, and you can remove this line and the one for `"github.com/influxdata/flux/stdlib/generate"`